src/unix/linux-core.c

```diff
@@ -606,7 +600,8 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
     rewind(statfile_fp);
     err = read_times(statfile_fp, numcpus, ci);
 
-  uv__close(statfile_fd);
+  while (fclose(statfile_fp) && errno != EINTR)
+    abort();
```

Comment:
This is not sound (but this gets obsoleted anyway when https://github.com/libuv/libuv/pull/747 lands, I think.)